instance/tests/models/test_utils.py

```diff
@@ -486,3 +486,35 @@ class DjangoResourceTest(SimpleResourceTestCase):
     Run the same tests as in SimpleResourceTestCase, but using DjangoResource.
     """
     make_resource = DjangoResource
+
+    def setUp(self):
+        self.make_resource.save = Mock()
```

Comment:
@bradenmacdonald 

> I think you could just add save = Mock() right in the definition of DjangoResource, rather than patching it or modifying it here.

This makes testing call counts for `save` in `test_mutator` a bit inconvenient: That test currently relies on having a fresh mock available, and it would be kind of awkward to use a "global" mock for the remaining tests and a separate one for `test_mutator` (or mess with the global mock directly in `test_mutator`). I kept the current approach for now.